dandi/tests/test_dandiapi.py

```diff
@@ -515,6 +515,7 @@ def test_remote_dandiset_json_dict(text_dandiset):
         "created": anys.ANY_AWARE_DATETIME_STR,
         "modified": anys.ANY_AWARE_DATETIME_STR,
         "contact_person": anys.ANY_STR,
+        "embargo_status": anys.ANY_STR,
```

Comment:
This isn't enough.  You also need to actually add the field definition to `RemoteDandiset` in `dandiapi.py`, in the manner of, e.g., [`contact_person`](https://github.com/dandi/dandi-cli/blob/b825018c8a2d36bf1f1b516a4959e753e979dcf0/dandi/dandiapi.py#L680-L683).